aikau/src/main/resources/alfresco/navigation/TreeStore.js

```diff
@@ -47,6 +47,49 @@ define(["dojo/_base/declare",
       filterPaths: null,
 
       /**
+       * This is a topic that can be published to request child data. This an alternative to the 
+       * [target]{@link module:alfresco/navigation/TreeStore#target} URL prefix that can also be configured.
+       *
+       * @instance
+       * @type {string}
+       * @default
+       * @since 1.0.39
+       */
+      publishTopic: null,
+
+      /**
+       * This is the payload that will be published to request child data when a
+       * [publishTopic]{@link module:alfresco/navigation/TreeStore#publishTopic} has been configured.
+       * 
+       * @instance
+       * @type {object}
+       * @default
+       * @since 1.0.39
+       */
+      publishPayload: null,
+
+      /**
+       * Indicates whether or not requests to get child data will be published globally using the 
+       * [publishTopic]{@link module:alfresco/navigation/TreeStore#publishTopic} that has been configured.
+       * 
+       * @instance
+       * @type {boolean}
+       * @default
+       * @since 1.0.39
+       */
+      publishGlobal: true,
```

Comment:
Because a) They weren't there when I started working on the code, and b) they do different things so the actual documentation is more useful to be duplicated and c) It's not using the actual mixin capabilities - there's no point in pulling in a mixin just for some variables